packages/edit-site/src/components/page-pages/index.js

```diff
@@ -40,8 +40,26 @@ const defaultConfigPerViewType = {
 	},
 };
 
+const PATH_TO_DATAVIEW_TYPE = {
+	'/pages': 'page',
```

Comment:
I think we don't need this here. We know we are in `/pages` path and we could just filter the default views based on the url param for the active one.